src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (true) {
+      case type === 'addProperties':
```

Comment:
nope, that's not how `switch/case` works. You should put type in first braces and then compare it's value by passing possible values to cases
```suggestion
    switch (type) {
      case  'addProperties':
```